IdokladSdk/IdokladSdk.csproj

```diff
@@ -75,4 +75,9 @@
     </PackageReference>
     <PackageReference Include="System.ComponentModel.Annotations" Version="4.7.0" />
   </ItemGroup>
+    <ItemGroup>
+        <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
+            <_Parameter1>$(MSBuildProjectName).UnitTests</_Parameter1>
+        </AssemblyAttribute>
+    </ItemGroup>
```

Comment:
Proc to tu je?

Reply:
Tohle je tu proto, aby se v unit testech dalo přistupovat i k typům a prvkům, které jsou internal.
Některé třídy máme totiž jako internal aby se k nim nemuseli psát ty dokumentační komentáře. Konkrétně jsem chtěl napsat test na ten ColorAttribute.